README-DEBUG.md

```diff
@@ -167,89 +167,53 @@ $3 = (*mut fn ()) 0x7f434efddbf9 <wasmldr::main+25>
 
 ### SGX
 
-Find the "shim" of the TEE. Normally this is `shim-sgx`:
-```console
-$ find target -wholename '*linux-musl/*/shim-sgx'
-target/debug/build/enarx-f0e8a07172ba3be9/out/internal/shim-sgx/x86_64-unknown-linux-musl/debug/shim-sgx
-```
+Before starting the debugging session, make sure you are in the project root
+directory.
+
+First, an appropriate shim binary for the debugging session must be found. An
+existing shim can be selected by issuing:
 
-Find the "exec" of the TEE. Normally this is `wasmldr`:
 ```console
-$ find target -wholename '*linux-musl/*/wasmldr'
-target/debug/build/enarx-f0e8a07172ba3be9/out/internal/wasmldr/x86_64-unknown-linux-musl/debug/wasmldr
+$ find target \
+  \( -perm -u=x -o -perm -g=x -o -perm -o=x \) \
+  -wholename "*debug*bin/enarx_shim_sgx*"
```

Comment:
This assumes that somebody has done `cargo build` and not `cargo build --release`, although maybe release builds aren't supported for debugging? Also, do we require people to have built with the `gdb` or `dbg` features enabled?

Reply:
Uf they are it would be a regression because you are not enforced to use `dbg` feature flag, when you are using `gdb` feature flag. Good remark, anyway, thank you.

Reply:
Added remark about the build.